engine/src/main/java/org/terasology/engine/network/internal/NetworkSystemImpl.java

```diff
@@ -1000,11 +1002,12 @@ private <T> Map<Class<? extends T>, Integer> applySerializationInfo(List<NetData
                     if (field != null) {
                         field.setId(info.getFieldIds().byteAt(i));
                     } else {
-                        logger.error("Server has unknown field '{}' on '{}'", info.getFieldName(i), info.getName());
+                        logger.atError().addArgument(info.getFieldName(i)).addArgument(() -> info.getName()).
+                                log("Server has unknown field '{}' on '{}'");
```

Comment:
`info.getFieldName(i)` referenced before (line 1000) and `info.getName()` too (line 996), so create and reference local variables